apps/condo/domains/banking/utils/validate/tin.utils.js

```diff
@@ -0,0 +1,86 @@
+/*
```

Comment:
Why tin utils is moved at this domain? Tin is not actually field of `BankAccount` model. It's also exist at `Organization`. Can it be moved to `common` domain for this reason?

Reply:
In the BankAccount model contain all requisites. I think it's more convenient to store all validators in one place

Reply:
I left original bankAccount model with original validator. We are going to change it when needed